### PR TITLE
Fix drawing glitch with chat vertical separator

### PR DESCRIPTION
**Pull request by paul-nameless**
#96
Implemented correct double width character count which fixes error with failed msg draw when emojies didn't count correctly.
Fixed glitch by drawing flags fristly and after that text message which made curses or terminal count correctly offset.

### CODE REVIEW

File: tg/utils.py [hunk 1]
```diff
@@ -180,9 +181,24 @@ def notify(
...
+    if real_len <= width:
+        return string
+
+    cur_len = 0
+    out_string = ""
+
+    for char in string:
+        cur_len += 2 if unicodedata.east_asian_width(char) in "WF" else 1
+        if cur_len < width:
+            out_string += char
```
Comment (lightme16) [inline tg/utils.py:200]: `else: break` ?

File: tg/utils.py [hunk 2]
```diff
@@ -180,9 +181,24 @@ def notify(
...
-    return s[: max(1, target_len - 1)]
+def string_len_dwc(string: str) -> int:
+    """Returns string len including count for double width characters"""
+    return sum(1 + (unicodedata.east_asian_width(c) in "WF") for c in string)
+
+
+def truncate_to_len(string: str, width: int) -> str:
+    real_len = string_len_dwc(string)
+    if real_len <= width:
+        return string
```
Comment (lightme16) [inline tg/utils.py:192]: this lines look redundant, do we really need them?

Comment (paul-nameless) [reply tg/utils.py:192]: Doesn't look redundant for me

Comment (paul-nameless) [reply tg/utils.py:192]: Suggest better alternative in other patch I would appreciate it.

**Pull request merged**